### DISCUSSION TITLE
How to center the name of a transition?

### DISCUSSION THREAD

**Discussion by kevonfernando** [gaphor/gaphor#1933] [Q&A] [answered]
Hello, 

I created a state machine and would like to center the text/name on the arrows/transitions. But I do not know how to do that. Does perhaps anybody here know how to achieve this?

**Comment by amolenaar** [comment, marked as answer]
Hi, transition names are supposed to be placed at the transition end, acoording to the UML specs. An optional guard condition is placed at the line center.

**Reply by kevonfernando** [reply]
That is to be honest a bit unsatisfying to hear, but I cannot really argue with the standard. Anyways, thank you for answering so quickly.
